website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,243 @@
+
+### End alignment 
+
+End alignment can be used when expressing numerical values with decimals as this aligns the decimal places vertically.
+Common examples of end alignment include:
+- Financial information and currency amounts.
+- Fractional and floating point values represented with decimals.
+
+![End alignment of content within a table](/assets/components/table/advanced-table/end-alignment-example.png)
+
+End alignment can also be used in the last column of an advanced table to:
+- Highlight a "more options" function pertaining to the content within a row.
+- As a means to visually "bookend" the row with content that is of a similar length, e.g., timestamps, TTL (time-to-live) values, dates.
+
+![End alignment example within a table with a date and more options](/assets/components/table/advanced-table/end-alignment-example-02.png)
+
+!!! Dont
+
+Don’t end align content that is variable in length. This can make the content more difficult to read by forcing an unnatural [reading pattern](/patterns/button-organization?tab=research#layout-and-reading-patterns).
+
+![End alignment with content that is variable in length](/assets/components/table/advanced-table/end-alignment-variable-length.png)
+!!!
+
+### Other alignment methods
+
+We don’t recommend center or justified alignment of content within a cell or advanced table. These alignment methods can result in the content being difficult to read, especially if it is variable in length.
+
+!!! Dont
+
+Don’t center header labels or cell content within a table.
+
+![Example of centered content within an Advanced Table](/assets/components/table/center-justified-alignment.png)
+!!!
+
+## Expandable rows
+
+### Striping 
+
+![Table striping examples](/assets/components/table/advanced-table/advanced-table-striping.png)
+
+Striping enhances readability by alternating row colors, making it easier to scan tabular data.
+- Non-Nested Advanced Tables: Striping starts with the second row, distinguishing it from the header.
+- Nested Advanced Tables:Child rows are automatically striped, while parent rows remain unstriped to visually reinforce hierarchy. This behavior cannot be disabled.
+
+Striping improves clarity for numerical or text-heavy data and ensures hierarchical relationships are visually distinct. For non-nested Advanced Tables, striping is optional and must be enabled with an argument. Always verify adequate color contrast when using striping.
+
+!!! Info
+
+Ensure that nested components within striped rows continue to maintain adequate color contrast with the striped background.
+!!!
+
+### Expandable rows
+
+Expandable rows allow users to expand or collapse additional content within the table structure.
+
+![Advanced Table expandable rows examples](/assets/components/table/advanced-table/expandable-rows.png) 
+
+### When to use
+
+- When organizing hierarchical data into parent-child relationships, allowing users to expand or collapse details as needed.
```

Comment:
[suggestion] This is a single item, I don't think it needs a bullet. Additionally, I think I remember you using this point about not navigating to a new page in the google doc which was a valid use case.

```suggestion
Use expandable rows when organizing hierarchical data into parent-child relationships, allowing users to expand or collapse details as needed without navigating to a new page.
```